docs/source/configuration/key_value_service_configs/postgres_key_value_service_config.rst

```diff
@@ -60,8 +60,64 @@ A minimal AtlasDB configuration for running against postgres will look like :
 
 For more details on the leader block, see :ref:`Leader Configuration <leader-config>`.
 
-Connection parameters
----------------------
+Hikari Connection parameters
+----------------------------
+
+We also provide the following options, which are mapped to `Hikari connection options <https://github.com/brettwooldridge/HikariCP#configuration-knobs-baby>`__:
```

Comment:
wow, that's an... unexpected URL (confirmed that the link is correct though!)

Reply:
yeah, it's very odd, now that you mention it... 😆